code/automated_test/python/automated_test_w_TCP.ipynb

```diff
@@ -0,0 +1,164 @@
+{
```

Comment:
Man kann in dieser Datei, so wie sie hier dargestellt ist, nicht wirklich sehen was passiert. Es sieht normalerweise recht cool aus. Vllt muss man direkt auf dem Branch gucken? So wie [hier.](https://github.com/bokeh/bokeh-notebooks/blob/master/gallery/linked_brushing.ipynb)